subscription-service/src/main/kotlin/com/egm/stellio/subscription/model/Subscription.kt

```diff
@@ -68,9 +65,9 @@ data class Subscription(
         else
             serializeWithoutSysAttrs(this.compact(contexts))
         return if (mediaType == JSON_LD_MEDIA_TYPE)
-            addContextToElement(serializedSubscription, contexts)
+            JsonLdUtils.addContextToElement(serializedSubscription, contexts)
         else
-            serializedSubscription
+            JsonLdUtils.removeContextToElement(serializedSubscription)
```

Comment:
creation time contexts should only be used for notifications. hence, they should not be part of the JSON(-LD) representation of a subscription (in other words, they can be ignored - @JsonIgnore)